android/assets/jsons/Civ V - Vanilla/Nations.json

```diff
@@ -392,7 +392,8 @@
 		"outerColor": [20,25,173],
 		"innerColor": [187,33,51],
 		"uniqueName": "Scholars of the Jade Hall",
-		"uniques": ["[+2 Science] from every specialist", "[+2 Science] from every [Great Improvement]"],
+        "uniques": ["[+2 Science] from every specialist", "[+2 Science] from every [Great Improvement]","Receive a tech boost when scientific buildings/wonders built in capital"],
+        "uniqueText": "+2 Science from all specialists and Great Improvement. Receive tech boost when scientific building/wonder is built in capital",
```

Comment:
Remove this. uniqueText is only for mods that want to do crazy things, the main nations should have text directly from the uiqus

Reply:
Removed uniqueText and corrected the korean unique to "Receive a tech boost when scientific buildings/wonders are built in capital" in commit below.